### PR TITLE
Added EDA (basic one - will work on it furter) and 2 linear models: SVC and LogReg

### PR DESCRIPTION
**Pull request by senyafeelsgood**

### CODE REVIEW

**Review by senyafeelsgood — COMMENTED** [review]
Added EDA, LogReg and SVC

**Comment by Savely-Prokhorov** [comment]
Супер, что пишешь текстовые комментарии к действиям в ноутбуке. На мой взгляд, все упоминания "давайте" перед некими производимы действиями лучше убрать. Будет легче читаться и в голове зазвучит более формально

Для `X_test `делаешь `fit_transform()`. Не надо так, замени на `transform()`

Предлагаю `clf` переименовать в одном месте в `logreg`, в другом в `svc`

Давай `coef_dict` выведем в виде датафрейма, отсортированного по значениям весов?

В самом конце: "Теперь давайте посмотрим, переобучилась ли наша выборка" -> "Теперь посмотрим, переобучилась ли наша _модель_"

Ещё странно в принципе считать accuracy. Выборка люто несбалансирована. Вместо accuracy предлагаю считать F1 Score. И я бы добавил ещё PR-AUC. Пожалуйста, при каждом эксперименте проверяй себя, что читаешь одни и те же метрики при прочих равных условиях. Можно вынести в `utils.py` или хотя бы просто выделить в ноутбуке небольшую функцию, которая будет печатать нужные метрики

Поправь, пожалуйста, то, на что я тебе указал. И подумаем, что делать дальше
